server/pm2_config.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Do we really have to have this configuration file?
Too much configuration files might be bad sign to us.